libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -584,6 +586,7 @@ const RC_Channel::LookupTable RC_Channel::lookuptable[] = {
     { AUX_FUNC::TURBINE_START, "Turbine Start"},
     { AUX_FUNC::FFT_NOTCH_TUNE, "FFT Notch Tuning"},
     { AUX_FUNC::MOUNT_LOCK, "MountLock"},
+    { AUX_FUNC::LOG_PAUSE, "LogPause"},
```

Comment:
I would label this "Pause Log" as otherwise it sounds like you are logging a pause

Reply:
done